pilot/pkg/xds/filters/filters.go

```diff
@@ -154,6 +160,20 @@ var (
 		})},
 	}
 
+	TCPClusterMx = &cluster.Filter{
+		Name: MxFilterName,
+		TypedConfig: util.MessageToAny(&udpa.TypedStruct{
```

Comment:
I think the filter config should be compiled-in now that's it's first-class.

Reply:
I thikn they are already first classed https://github.com/istio/proxy/pull/3229. It was a mistake that they were not removed from inbound in this PR: https://github.com/istio/istio/pull/31226/files